python/hail/dataset.py

```diff
@@ -3305,14 +3305,15 @@ def min_rep(self, left_aligned=False):
         return VariantDataset(self.hc, self._jvds.minRep(left_aligned))
 
     @handle_py4j
-    @require_biallelic
+    #@require_biallelic
     @record_method
     @typecheck_method(scores=strlike,
                       loadings=nullable(strlike),
                       eigenvalues=nullable(strlike),
                       k=integral,
+                      entry_to_double=nullable(strlike),
                       as_array=bool)
-    def pca(self, scores, loadings=None, eigenvalues=None, k=10, as_array=False):
+    def pca(self, scores, loadings=None, eigenvalues=None, k=10, entry_to_double=None, as_array=False):
```

Comment:
I don't really like `entry_to_double` as a name, esp. since the expression doesn't have to be a double, just convertible to `Double`.  How about `entry_expr`?  That matches our other commands a little better.